addons/mail/static/src/messaging/component/message/message.js

```diff
@@ -264,7 +201,10 @@ class Message extends Component {
      * @param {integer} [offset=0]
      * @return {boolean}
      */
-    isBottomVisible({ offset=0 }={}) {
+    isBottomVisible({ offset=0 } = {}) {
```

Comment:
object param for only one is confusing

Reply:
`.isBottomVisible(100)` is not clear compared to `.isBottomVisible({ offset: 100 })`. I don't really like positional arguments as integers/boolean/string when its not straightforward like here

Reply:
```
const offset = 100;
.isBottomVisible(offset)
```
?

Reply:
This is a problem with unnamed string and integers in general, and I think we should avoid them.

Reply:
Don't like naming local variable neither.
I think it makes sense for signature of function to be explicit, i.e. by naming this parameter.